for_workspace/make_build.bzl

```diff
@@ -0,0 +1,47 @@
+""" Rule for building GNU Make from sources. """
```

Comment:
So in the equivalent cmake_build.bzl, you are still using the make from the system not this one. That's WAI isn't it?

Reply:
Well, I tried to update that to use the Make toolchain, but I couldn't get the dependencies right. I think it's worth trying that once more, but for the purpose of this pull request, that can considered an additional feature.

Especially given that we cannot bootstrap Make from absolutely nothing: we still need basic build tools like `automake` and `texinfo`. (Maybe bootstrap-from-nothing is also a feature to consider, but for me it's not particularly interesting: I mostly added the bootstrapped make for completeness. The main feature is to have the Make toolchain so that it could be swapped with another toolchain.)